packages/sources/svelte/src/components/structure/VtmnDivider/VtmnDivider.svelte

```diff
@@ -28,6 +30,12 @@
    */
   export { className as class };
 
+  onMount(async () => {
```

Comment:
You can set the `uuid` by default on the export let 

```
export let labelId = uuid();
```

If the user define the `labelId`, it takes the defined one :)